src/bio/smd/tests/smd_ut.c

```diff
@@ -479,7 +481,7 @@ ut_dev_replace(void **state)
 	/* Verify blob IDs after device replace */
 	d_list_for_each_entry_safe(pool_info, tmp_pool, &pool_list, spi_link) {
 		for (i = 0; i < 3; i++) {
-			rc = smd_pool_get_blob(pool_info->spi_id, i, &blob_id);
+			rc = smd_pool_get_blob(pool_info->spi_id, i, SMD_DEV_TYPE_DATA, &blob_id);
 			assert_rc_equal(rc, 0);
 			if (i == 0)
 				assert_int_equal(blob_id, 555);
```

Comment:
We need unit tests for WAL & meta devices too.

Reply:
That'll be great if you make it in this PR, but I'm fine with making it in follow-on PR as well. Thanks.

Reply:
Working on that

Reply:
Updated smd_ut to include META/WAL target types